### PR TITLE
scalafmt: config-style arguments

### PR DESCRIPTION
**Pull request by raboof**

### CODE REVIEW

File: akka-distributed-data/src/test/scala/akka/cluster/ddata/WriteAggregatorSpec.scala [hunk 1]
```diff
@@ -213,7 +213,10 @@ class WriteAggregatorSpec extends AkkaSpec(s"""
       val t = timeout / 5 - 50.milliseconds.dilated
       import system.dispatcher
       Future.sequence {
-        Seq(Future { testProbes(nodeC).expectNoMessage(t) }, Future { testProbes(nodeD).expectNoMessage(t) })
+        Seq(
+          Future { testProbes(nodeC).expectNoMessage(t) },
+          Future { testProbes(nodeD).expectNoMessage(t) }
+        )
```
Comment (raboof) [inline akka-distributed-data/src/test/scala/akka/cluster/ddata/WriteAggregatorSpec.scala]: This is the main motivation for this PR:
* with `optIn.configStyleArguments = false`, scalafmt insists to put this all on one line.
* without `optIn.configStyleArguments = false`, scalafmt allows both variations

It causes a lot of other changes (so possibly merge conflicts) though, so not sure if it's worth it.

Comment (patriknw) [reply akka-distributed-data/src/test/scala/akka/cluster/ddata/WriteAggregatorSpec.scala]: what does "allows both variations" mean? Did you manually add a newline before the `)` and then it keeps it multi-line, or how do one decide?

File: akka-distributed-data/src/main/scala/akka/cluster/ddata/protobuf/ReplicatorMessageSerializer.scala [hunk 2]
```diff
@@ -334,7 +337,8 @@ class ReplicatorMessageSerializer(val system: ExtendedActorSystem)
         val fromSeqNr = e.getFromSeqNr
         val toSeqNr = if (e.hasToSeqNr) e.getToSeqNr else fromSeqNr
         e.getKey -> Delta(dataEnvelopeFromProto(e.getEnvelope), fromSeqNr, toSeqNr)
-      }.toMap)
+      }.toMap
+    )
```
Comment (patriknw) [inline akka-distributed-data/src/main/scala/akka/cluster/ddata/protobuf/ReplicatorMessageSerializer.scala]: I'm not sure I like the extra lines in all places

File: akka-distributed-data/src/test/scala/akka/cluster/ddata/WriteAggregatorSpec.scala (hunk 1 above)
Comment (raboof) [reply akka-distributed-data/src/test/scala/akka/cluster/ddata/WriteAggregatorSpec.scala]: Yes: I manually made this change, scalafmt leaves both the original and the new style intact.

Comment (patriknw) [reply akka-distributed-data/src/test/scala/akka/cluster/ddata/WriteAggregatorSpec.scala]: ok, but based on the other changes it looks like the trigger is a newline after the `(`, and I'm not quite sure I like that extra newline before `)` in all such places

Comment (raboof) [reply akka-distributed-data/src/test/scala/akka/cluster/ddata/WriteAggregatorSpec.scala]: Indeed it looks like it'll either put everything on one line or insists on also giving the `)` its own line. I agree that it doesn't work out nicely in all places.

**Pull request closed without merging**